src/com/thunder_cut/graphics/ui/drawing/CanvasPixelInfo.java

```diff
@@ -24,12 +24,15 @@ public CanvasPixelInfo(int width, int height, Color backgroundColor) {
         }
     }
 
-    public void toBufferedImage(BufferedImage image) {
+    public BufferedImage toBufferedImage(int width, int height) {
```

Comment:
Use field width, height.    
There is no reason using parameter width and height.

Reply:
The field "width" and "height" are set by constructor.